File: Modul_5/Soal_3/PRAK503-2310817210015-MuhammadBukhariFitri.py
def maksimal(a, b):
    # Lengkapi Function ini
    if(a < b) :
        max = b
    else :
        max = a
    return max

def minimal(a, b):
    # Lengkapi Function ini
    if(a > b) :
        min = b
    else :
        min = a
    return min

batas = 0
maks = -100000
minim = 100000
bilangan = int(input())
while(batas < bilangan):
    nilai = map(int, input().split())
    for i in nilai :
        maks = maksimal(maks, i)
        minim = minimal(minim, i)
        batas += 1
print(f"{maks} {minim}") 

File: Modul_5/Soal_3/PRAK503-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int maksimal(int a, int b){
    // Lengkapi Function ini
    int max;
    if(a < b) {
        max = b;
    }
    else {
        max = a;
    }
    return max;
}

int minimal(int a, int b){
    // Lengkapi Function ini
    int min;
    if(a > b) {
        min = b;
    }
    else {
        min = a;
    }
    return min;
}

int main(){
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;
    scanf("%d", &bilangan);
    while(batas < bilangan){
        int nilai;
        scanf("%d", &nilai); 
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
        }
    printf("%d %d",maks,minim); 
}